### PR TITLE
create-about-file

### PR DESCRIPTION
**Pull request by ehumph**
This creates the about.md for the shiny app. It includes a basic background to the project and who exactly we are.

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+For this project, we're examining how the soil microbes vary based on soil characteristics and land use. We collect soil samples for both metagenomic and chemical analysis, and then use the data to look for patterns. This project offers students an opportunity to participate in a real research project (from sample collection to data analysis) as well as training in genomics methods, training in data science, and training in using cloud computing resources like AnVIL.
```
Comment (avahoffman) [inline about.md:5]: Maybe we want to be more general here to keep it within NHGRI scope, and just say this project aims to be at the intersection of genomics, ds, cloud computing, and education. Then get into the details below?

File: about.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+## What is genomics?
+
+Genomics broadly refers to the study of genomes, which are an organism's complete set of DNA. This includes both genes and non-coding regions of DNA. Traditional genomics involves sequencing and analyzing the genome of individual species.
+
+Metagenomics expands genomics to look at the collective genomes of entire communities of organisms in an environmental sample, like soil. It allows researchers to study not just the genes of culturable or isolated organisms, but the entirety of genetic material present in a given environment. By using genomic techniques to survey the soil microbes, we can identify everything in the soil, including microbes that no one has identified before.
```
Comment (avahoffman) [inline about.md:11]: Do we want to point that we will sorta do both? some long read sequencing will involve genome assembly

File: about.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+## Why soil microbes?
+
+It can be challenging to include undergraduates in human genomic and health research, especially in a classroom context. Both human genetic data and human health data are protected data, which limits the sort of information students can access without undergoing specialized ethics training. However, the same sorts of data cleaning and analysis methods used for human genomic data are also used for microbial genomic data, which does not have the same sort of legal protections as human genetic data. This makes it ideal for training undergraduate students at the beginning of their careers and can be used to prepare students for future research in human genomics and health [Jurkowski et al. 2017][5]. Additionally, the microbes in the soil can have big impacts on our health [Brevik and Burgess 2004][6].
```
Comment (avahoffman) [inline about.md:23]: jurkowski is 2007 - the cbe website has a typo (crazy!)

**Review by avahoffman — APPROVED** [review]
This is great!!!! 🤩🤩🤩🤩🤩

- [ ] Can we move this to the `www` directory
- [ ] We'll need some code in `R/ui.R` to make this visible. Would you like me to do that?
- [ ] Think pictures might be nice? We could add that in a follow up PR.

**Comment by ehumph** [comment]
> This is great!!!! 🤩🤩🤩🤩🤩
> 
> Love the "about us" -- I think people are very curious :)
> 
> * [ ]  Can we move this to the `www` directory
> * [ ]  We'll need some code in `R/ui.R` to make this visible. Would you like me to do that?
> * [ ]  Think pictures might be nice? We could add that in a follow up PR.

I will move this into the appropriate directory - if you can put together the code to make it visible, that would be awesome! And yes, more pictures are always better (:

File: about.md (hunk 3 above)
Comment (ehumph) [reply about.md]: That is insane to me

**Comment by avahoffman** [comment]
> > This is great!!!! 🤩🤩🤩🤩🤩
> > Love the "about us" -- I think people are very curious :)
> > 
> > * [ ]   Can we move this to the `www` directory
> > * [ ]   We'll need some code in `R/ui.R` to make this visible. Would you like me to do that?
> > * [ ]   Think pictures might be nice? We could add that in a follow up PR.
> 
> I will move this into the appropriate directory - if you can put together the code to make it visible, that would be awesome! And yes, more pictures are always better (:

Sounds good, I'll tackle that!

**Pull request merged**